openEHR_RM/es/openehr_terminology.xml

```diff
@@ -0,0 +1,322 @@
+<terminology name="openehr" language="es">	
+	<codeset issuer="openehr" openehr_id="compression algorithms" external_id="openehr_compression_algorithms">
+		<code value="compresión"/>
+		<code value="deflación"/>
+		<code value="gzip"/>
+		<code value="zlib"/>
+		<code value="otro"/>
+	</codeset>
+	<codeset issuer="openehr" openehr_id="integrity check algorithms" external_id="openehr_integrity_check_algorithms">
+		<code value="SHA-1"/>
+		<code value="SHA-256"/>
+	</codeset>	
+	<codeset issuer="openehr" openehr_id="normal statuses" external_id="openehr_normal_statuses">
```

Comment:
These normal_statuses are based on the EN words High, Normal, Low. In Spanish I would expect Alto, Normal, Bajo. I don't know is A/N/B appear in Spanish language lab results - and this particular set of codes might not be used much anyway, so it might not matter.